source/main/modules/window-manager/index.ts

```diff
@@ -75,6 +77,8 @@ export default class WindowManager {
   private _fileLock: boolean
   private _persistTimeout: ReturnType<typeof setTimeout>|undefined
   private _beforeMainWindowCloseCallback: Function|null
+  // Add tray to MacOS
```

Comment:
This comment isn't accurate because the variable is for all platforms. I suggest delete the comment.